### PR TITLE
Jtm view details reconfig

### PR DESCRIPTION
**Pull request by BigChiefSmidgeums**

### CODE REVIEW

File: app/controllers/dashboard/protocols_controller.rb [hunk 1]
```diff
@@ -205,12 +205,6 @@ def display_requests
     render json: data
   end
 
-  def view_details
```
Comment (kyle-glick) [inline app/controllers/dashboard/protocols_controller.rb]: I think there was a :view_details symbol in one of the before_filters in this controller. `before_filter find_protocol`. Can you remove it?

File: spec/controllers/protocols/get_show.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright © 2011-2016 MUSC Foundation for Research Development
```
Comment (kayla-glick) [inline spec/controllers/protocols/get_show.rb]: Minor, but spec should be named get_show_spec.rb to match naming conventions

File: spec/factories/protocol.rb [hunk 3]
```diff
@@ -124,7 +130,7 @@
     end
 
     factory :protocol_without_validations, traits: [:without_validations]
-    factory :study_without_validations, traits: [:without_validations, :study]
+    factory :protocol_study_without_validations, traits: [:without_validations, :study]
```
Comment (kayla-glick) [inline spec/factories/protocol.rb]: Should rename to study_without_validations to be consistent with project_without_validations

File: spec/factories/protocol.rb [hunk 4]
```diff
@@ -133,9 +139,13 @@
     factory :protocol_federally_funded, traits: [:funded, :federal]
     factory :protocol_with_sub_service_request_in_cwf, traits: [:with_sub_service_request_in_cwf, :funded, :federal]
     factory :study_with_blank_dates, traits: [:study, :pending, :blank_funding_start_dates, :blank_start_and_end_dates]
+    factory :protocol_invalid_study_with_questions, traits: [:without_validations, :study, :with_study_type_questions_group]
```
Comment (kayla-glick) [inline spec/factories/protocol.rb]: See previous comment for naming convention. What makes this invalid? Should that be without_validations?

File: spec/factories/protocol.rb [hunk 5]
```diff
@@ -133,9 +139,13 @@
...
   end
 
   factory :study, parent: :protocol, class: 'Study' do
     type "Study"
   end
-end
+
+  factory :study_without_validations, parent: :protocol_study_without_validations, class: 'Study'
```
Comment (kayla-glick) [inline spec/factories/protocol.rb]: Why are you aliasing other factories without new traits?

**Review by kayla-glick — CHANGES_REQUESTED** [review]

**Review by kayla-glick — APPROVED** [review]

**Review by Stuart-Johnson — APPROVED** [review]

**Review by jwiel86 — APPROVED** [review]

**Pull request merged**